content/en/blog/2023/integrations.md

```diff
@@ -0,0 +1,37 @@
+---
+title: OpenTelemetry Integrations Welcome!
+linkTitle: Integrations Welcome!
+date: 2023-11-03
```

Comment:
Let's set this to be some time later this week.

Reply:
Done. Publication date set to 11/08.